### PR TITLE
[MISC] Small refactoring of prefixes

### PR DESCRIPTION
**Pull request by smehringer**
This will make it easier for raptor to parse layout files in case we decide to alter the look of the layout header.

### CODE REVIEW

File: include/chopper/layout/hierarchical_binning.hpp [hunk 1]
```diff
@@ -294,9 +294,9 @@ class hierarchical_binning
         if (data->output_buffer->tellp() == 0) // beginning of the file
         {
             if (config.debug)
-                *data->output_buffer << "#FILES\tBIN_INDICES\tNUMBER_OF_BINS\tEST_MAX_TB_SIZES\tSCORE\tCORR\tT_MAX" << std::endl;
+                *data->output_buffer << prefix::header <<"FILES\tBIN_INDICES\tNUMBER_OF_BINS\tEST_MAX_TB_SIZES\tSCORE\tCORR\tT_MAX" << std::endl;
```
Comment (eseiler) [inline include/chopper/layout/hierarchical_binning.hpp:297]:
```suggestion
                *data->output_buffer << prefix::header << "FILES\tBIN_INDICES\tNUMBER_OF_BINS\tEST_MAX_TB_SIZES\tSCORE\tCORR\tT_MAX" << std::endl;
```

File: include/chopper/layout/hierarchical_binning.hpp [hunk 2]
```diff
@@ -294,9 +294,9 @@ class hierarchical_binning
...
             else
-                *data->output_buffer << "#FILES\tBIN_INDICES\tNUMBER_OF_BINS" << std::endl;
+                *data->output_buffer << prefix::header <<"FILES\tBIN_INDICES\tNUMBER_OF_BINS" << std::endl;
```
Comment (eseiler) [inline include/chopper/layout/hierarchical_binning.hpp:299]:
```suggestion
                *data->output_buffer << prefix::header << "FILES\tBIN_INDICES\tNUMBER_OF_BINS" << std::endl;
```

File: include/chopper/prefixes.hpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string_view>
+
+namespace chopper::prefix
+{
```
Comment (eseiler) [inline include/chopper/prefixes.hpp:6]:
```suggestion
namespace chopper::prefix
{

```

File: include/chopper/prefixes.hpp [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+constexpr std::string_view header{"#"};
+
+constexpr std::string_view hibf{"HIGH_LEVEL_IBF"};
+
+constexpr std::string_view merged_bin{"MERGED_BIN"};
+
+constexpr std::string_view split_bin{"SPLIT_BIN"};
+
+} // namespace chopper
```
Comment (eseiler) [inline include/chopper/prefixes.hpp:15]:
```suggestion
} // namespace chopper::prefix
```

File: include/chopper/prefixes.hpp [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string_view>
+
+namespace chopper::prefix
+{
+constexpr std::string_view header{"#"};
+
+constexpr std::string_view hibf{"HIGH_LEVEL_IBF"};
```
Comment (eseiler) [inline include/chopper/prefixes.hpp:9]:
```suggestion
constexpr std::string_view high_level{"HIGH_LEVEL_IBF"};
```
?

**Review by eseiler — CHANGES_REQUESTED** [review]

File: src/chopper_layout.cpp [hunk 6]
```diff
@@ -290,7 +290,7 @@ int execute(seqan3::argument_parser & parser)
 
     // brief Write the output to the result file.
     std::ofstream fout{config.output_filename};
-    fout << "#" << chopper::hibf_prefix << " max_bin_id:" << max_hibf_id << '\n';
+    fout << prefix::header << prefix::hibf << " max_bin_id:" << max_hibf_id << '\n';
```
Comment (smehringer) [inline src/chopper_layout.cpp:293]:
```suggestion
    fout << prefix::header << prefix::high_level << " max_bin_id:" << max_hibf_id << '\n';
```

**Review by eseiler — APPROVED** [review]

**Pull request merged**